browser/src/control/Control.MobileTopBar.js

```diff
@@ -205,9 +205,7 @@ L.Control.MobileTopBar = L.Control.extend({
 				toolbarDownButtons.forEach(function(id) {
 					toolbar.enable(id);
 				});
-				toolbar.hide('PermissionMode');
-				toolbar.hide('after-PermissionMode');
-				$('#tb_actionbar_item_before-PermissionMode').width('');
+				$('#MobilePermissionMode').hide();
```

Comment:
note: please try to avoid using jQuery in the new code if possible.
For example you can use:
`document.querySelector('#MobilePermissionMode')`
element.style.display = 'none' for hide
element.style.display = '' for show

Reply:
Updated. Thanks @eszkadev 🙇🏼